guides/source/rails_on_rack.md

```diff
@@ -80,6 +80,18 @@ To find out more about different `rackup` options, you can run:
 $ rackup --help
 ```
 
+#### Prefix your app using `config.relative_url_root`
+
+When using `rackup` and you are deploying your app to a subdirectory, you need to use `map` to define the root url to your app when generating the `Rack::URLMap` structure.
+
+```ruby
+# Rails.root/config.ru
+
+map RelativeRoot::Application.config.relative_url_root || '/' do
```

Comment:
`RelativeRoot` might be confusing here, `TestApp` would probably be more appropriate

Reply:
@arianf Fear not! Do you have opinions re this comment: https://github.com/rails/rails/pull/42303/files#r641911853